PYME/Acquire/PYMEAcquire.py

```diff
@@ -66,6 +66,9 @@ def setup_logging(default_level=logging.DEBUG):
     else:
         logging.basicConfig(level=default_level)
 
+    # we want to somehow supress the level of matplotlib messages below debug level
+    logging.getLogger('matplotlib').setLevel(logging.ERROR) #clobber unhelpful matplotlib debug messages
```

Comment:
Note that I tried the `dsviewer` approach of just silencing selected `matplotlib` packages, i.e.

```python
logging.getLogger('matplotlib.font_manager').setLevel(logging.ERROR) #clobber unhelpful matplotlib debug messages
logging.getLogger('matplotlib.backends.backend_wx').setLevel(logging.ERROR)
```
but this still left a lot of debug garbage, so resorted to silencing at the top level. This works now fine, one can discuss if setting to `logging.ERROR` or `logging.WARN`.

Reply:
hmm - I think the targeted silencing was initially because the top-level silencing didn't work  - maybe that's now fixed in matplotlib.